tests/Report/CheckstyleReporterTest.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Report;
+
+use GeckoPackages\PHPUnit\Constraints\XML\XMLMatchesXSDConstraint;
+use PhpCsFixer\Report\CheckstyleReporter;
+use PhpCsFixer\Report\ReportSummary;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author Kévin Gomez <contact@kevingomez.fr>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Report\CheckstyleReporter
+ */
+final class CheckstyleReporterTest extends TestCase
+{
+    /**
+     * @var CheckstyleReporter
+     */
+    private $reporter;
+
+    /**
+     * "checkstyle" XML schema.
+     *
+     * @var string
+     */
+    private $xsd;
+
+    protected function setUp()
+    {
+        $this->reporter = new CheckstyleReporter();
+        $this->xsd = file_get_contents(__DIR__.'/../../doc/checkstyle.xsd');
+    }
+
+    /**
+     * @covers \PhpCsFixer\Report\CheckstyleReporter::getFormat
```

Comment:
The test class already has a `@covers` annotation, isn't this redundant?

Reply:
ref #3133